files.php

```diff
@@ -295,11 +296,31 @@ function find_parameters($path){
 					}
 				}
 
+
+                function make_webstartlink($path,$dir){
+                    // BAM files
+                    $url_short = str_replace('/storage/', '', $dir);
+                    $bam_web_link = "";
+                    if(substr($path, -4) == '.bam'){
+                       $bam_web_link = '<BR>><a href="http://www.broadinstitute.org/igv/projects/current/igv.php?sessionURL=http://ctr-bfx.pdn.private.cam.ac.uk/' . $url_short . $path . '&genome=hg38">[Launch IGV]</a><FONT style="font-size:6pt">(Broken in Chrome)</FONT>';
```

Comment:
Cool! Can we move the specific URL bits out into config variables again though please, to generalise?